protocol.md

```diff
@@ -447,8 +464,9 @@ enum GroupMemberFlags {
 ```
 
 ### **0x19** ArchiveChannel ![networkDuplex] ###
-This changes a channel's archive mode which is necessary to avoid sending messages to a deleted or blocked channel.
-In contrast to deleted channels, clients can still send packets like `GroupChannelKeyNotify` over blocked channels.
+This packet changes a channel's archive mode which is necessary to avoid sending messages to a deleted or blocked channel.
+Only direct channels can be blocked. They are managed by the server and in contrast to deleted channels, clients can still send any packets except _ChatMessage_ over blocked channels.
```

Comment:
Why can I send non-ChatMessage packets to a _blocked_ channel?

Reply:
Also, does that include Message Override? Because if that wasn't included, I could edit messages in blocked channels.